elasticsearch/run-elasticsearch.sh

```diff
@@ -7,30 +7,67 @@ if [[ -z $STACK_VERSION ]]; then
   exit 1
 fi
 
+MAJOR_VERSION=`echo ${STACK_VERSION} | cut -c 1`
+
 docker network create elastic
 
 NODES=${NODES-1}
 for (( node=1; node<=$NODES; node++ ))
 do
   port=$((9200 + $node - 1))
-  docker run \
-    --rm \
-    --env "node.name=es${node}" \
-    --env "cluster.name=docker-elasticsearch" \
-    --env "cluster.initial_master_nodes=es1" \
-    --env "discovery.seed_hosts=es1" \
-    --env "cluster.routing.allocation.disk.threshold_enabled=false" \
-    --env "bootstrap.memory_lock=true" \
-    --env "ES_JAVA_OPTS=-Xms1g -Xmx1g" \
-    --env "xpack.security.enabled=false" \
-    --env "xpack.license.self_generated.type=basic" \
-    --ulimit nofile=65536:65536 \
-    --ulimit memlock=-1:-1 \
-    --publish "${port}:9200" \
-    --detach \
-    --network=elastic \
-    --name="es${node}" \
-    docker.elastic.co/elasticsearch/elasticsearch:${STACK_VERSION}
+  if [ "x${MAJOR_VERSION}" == 'x5' ]; then
+    docker run \
+      --rm \
+      --env "node.name=es${node}" \
+      --env "cluster.name=docker-elasticsearch" \
+      --env "cluster.routing.allocation.disk.threshold_enabled=false" \
+      --env "bootstrap.memory_lock=true" \
+      --env "ES_JAVA_OPTS=-Xms1g -Xmx1g" \
+      --env "xpack.security.enabled=false" \
+      --ulimit nofile=65536:65536 \
+      --ulimit memlock=-1:-1 \
+      --publish "${port}:9200" \
+      --detach \
+      --network=elastic \
+      --name="es${node}" \
+      docker.elastic.co/elasticsearch/elasticsearch:${STACK_VERSION}
```

Comment:
You can remove this check, as we don't support Elasticsearch 5 :)

Reply:
That's right.
fixed.
https://github.com/miyataka/elastic-github-actions/actions/runs/109452371